app/src/main/java/com/lagradost/cloudstream3/utils/UIHelper.kt

```diff
@@ -123,6 +127,20 @@ object UIHelper {
         )
     }
 
+    fun clipboardHelper(label: String, text: CharSequence) {
```

Comment:
This should be a UiText instead, to allow for both Strings and Stringres

Reply:
wow didnt knew this exists